### ISSUE TITLE
Secret tip for removing zero-padding on unix-like systems

### ISSUE THREAD

**Issue by patricksurry** [mccutchen/strftime.org#11]
This page is a great idea, I use it semi-regularly :-)

One handy tip that might be worth adding is that on unix-like systems, %-d will give you the day of month without leading zeros.  Not portable to Windows though.

**Comment by mccutchen** [comment]
@patricksurry whoa, I didn't know about that little trick!  I think it might make some good sense to have a separate "tips & tricks" section of some sort.  Feel like submitting a pull request?

**Comment by patricksurry** [comment]
Sure, I wasn't clear if you were just pulling docs from the python trunk or
had your own local stuff, but I'll take a look.

**Comment by mccutchen** [comment]
I'm doing it in just about the dumbest way possible: fetching the current docs from http://docs.python.org/2/library/datetime.html and parsing the info I want out of the table on that page.

It all happens in [build.py][1].  Reproducing this locally should be as easy as:

```bash
git clone git@github.com:mccutchen/strftime.org.git
cd strftime.org
pip install -r requirements.txt
fab build
open dist/index.html
```

[1]: https://github.com/mccutchen/strftime.org/blob/master/build.py

**Comment by mccutchen** [comment]
Fixed by #12
